src/app/pages/AccountDetailsPage/AccountTokenTransfersCard.tsx

```diff
@@ -23,19 +23,17 @@ export const AccountTokenTransfersCard: FC<AccountDetailsContext> = ({ scope, ad
       </LinkableDiv>
       <CardContent>
         <ErrorBoundary light={true}>
-          <AccountTokenTransfers scope={scope} address={address} />
+          <AccountTokenTransfers scope={scope} address={address} account={account} />
         </ErrorBoundary>
       </CardContent>
     </Card>
   )
 }
 
-const AccountTokenTransfers: FC<AccountDetailsContext> = ({ scope, address }) => {
+const AccountTokenTransfers: FC<AccountDetailsContext> = ({ scope, address, account }) => {
   const { t } = useTranslation()
-
   const { isLoading, isFetched, results } = useTokenTransfers(scope, address)
 
-  const { account } = useAccount(scope, address)
```

Comment:
Not sure if this was by design, but these useAccount calls are duplicated, so I just moved the account to the OutletContext

Reply:
oh no some of these changes are worse. https://94637592.oasis-explorer.pages.dev/mainnet/sapphire/address/0x9BC6E51BCc4A23dfF5ede63ed35BB0cFFCB3468c/tokens/erc-20#tokens shows `This account holds no ERC-20 tokens` instead of a loading state